chains/v13/chains.json

```diff
@@ -298,6 +298,15 @@
                     "type": "github",
                     "url": "https://raw.githubusercontent.com/novasamatech/nova-utils/master/crowdloan/westend.json"
                 }
+            ],
+            "governance": [
```

Comment:
if we add polkassembly API, then we need to add gov type. Afaik they don't have any data now in their API